frontend/src/styles/index.scss

```diff
@@ -23,6 +23,14 @@ body {
     Button {
       margin: 1%;
     }
+
+    .name {
+      color: #4FAF4E
+    }
+
+    .description {
+      color: #484848
+    }
```

Comment:
Another CSS LEGEND in the making. Some comments about the CSS atm though is that generally the index.scss/index.css file is used for more general classes/styling. Classes that are very broad and can be reused across multiple components are kept at this level as a result (some examples are like .text-center, which can apply in a lot of different cases in many different css files and classes).

For more specific styling or CSS targets that apply specifically to one component, another SCSS file should be made. So in this case you would probably want another SCSS file called notification.scss.

Also SCSS has a very neat feature of being able to define variables and use/import them throughout the rest of the SCSS files, so I would consider looking into making a variables.scss file that defines all of your colors and then importing them in other SCSS files. If you need some examples with that, I did a little bit of that in the dashboard-styling branch, so feel free to check that out :))